eventmesh-admin/eventmesh-admin-rocketmq/src/main/java/org/apache/eventmesh/admin/rocketmq/request/TopicCreateRequest.java

```diff
@@ -30,16 +30,13 @@ public class TopicCreateRequest {
 
     @JsonCreator
     public TopicCreateRequest(@JsonProperty("name") String topic) {
-        super();
         this.name = topic;
```

Comment:
Maybe, using the same variable name for topic and name would improve readability.

Reply:
I'm not sure. Maybe the author plan to add more fields to this class in the future.
Will it be more readable than now when more fields are added to this class?

Reply:
change the `name` to `topic` is better